free-2.md

```diff
@@ -58,12 +72,17 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think this would be a systems-heavy project. A lot of back-end work would have to be done, in order to create a list of wines that are suggested, and to create a way to rank wines based on specific attributes of various foods.
```

Comment:
I agree. Although this is a cool project, I feel like the biggest part of this task is database work and not language design.